Registry/Registry.NMM

```diff
@@ -1775,6 +1774,9 @@ package   wdm7scheme    mp_physics==26               -             moist:qv,qc,q
 package   p3_1category    mp_physics==50               -             moist:qv
 package   p3_1category_nc mp_physics==51               -             moist:qv
 package   p3_2category    mp_physics==52               -             moist:qv
+#--------------------------------------ntu3m------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
@tzuchin12 
Similarly here, you can remove the 
```
#---------for ntu3m---
```
lines for the package statements

Reply:
Modified as suggested.